packages/vm/src/state/stateManager.ts

```diff
@@ -513,4 +511,20 @@ export default class DefaultStateManager extends BaseStateManager implements Sta
     }
     return false
   }
+
+  /**
+   * Gets the account associated with `address`, modifies the given account
+   * fields, then saves the account into state. Account fields can include
+   * `nonce`, `balance`, `stateRoot`, and `codeHash`.
+   * @param address - Address of the account to modify
+   * @param accountFields - Object containing account fields and values to modify
+   */
+  async modifyAccountFields(address: Address, accountFields: AccountFields): Promise<void> {
+    const account = await this.getAccount(address)
+    account.nonce = accountFields.nonce ?? account.nonce
+    account.balance = accountFields.balance ?? account.balance
+    account.stateRoot = accountFields.stateRoot ?? account.stateRoot
+    account.codeHash = accountFields.codeHash ?? account.codeHash
+    await this.putAccount(address, account)
+  }
```

Comment:
This can be placed in the `BaseStateManager` if I am not mistaken.

Reply:
Ooh ok, sure, I can move it

Reply:
Thanks! 😄

Doing the `develop` rebase (test) right now.